src/index.html

```diff
@@ -9,10 +9,74 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Link to the SCSS file is incorrect. You should link to the compiled CSS file instead. For example, if your SCSS compiles to a `main.css` file, the link should be `<link rel="stylesheet" href="styles/main.css" />`.